lib/liquid/tags/include.rb

```diff
@@ -44,7 +43,15 @@ def parse(tokens)
 
     def render(context)
       partial = load_cached_partial(context)
-      variable = context.evaluate(@variable_name)
+
+      template_name = context.evaluate(@template_name_expr)
+      context_variable_name = template_name.split('/'.freeze).last
```

Comment:
`Expression.parse` takes care of turning quoted strings into strings (if it was unquoted it'd return a `VariableLookup` and `context.evaluate` would look up the variable on the previous line).